security/README.md

```diff
@@ -0,0 +1,53 @@
+Security
+========
+
+A guide for practicing safe web.
+
+Think
+-----
+
+Security is important, and you can't practice these guidelines without
+understanding them. Question each guideline before following it yourself and
+start a discussion whenever you're in doubt.
+
+Using Passwords
+---------------
+
+* Use a unique password for every account you create.
+* Use a tool like [pwgen](https://github.com/jbernard/pwgen) or
+  [1password](https://agilebits.com/onepassword) to generate random passwords.
+* Use a tool like GnuPG to encrypt passwords if you need to share them with
+  somebody.
+
+Accepting Passwords
+-------------------
+
+* Don't store passwords in plain text.
+* Don't hash passwords using a reversible cipher.
+* Don't hash passwords using a broken cipher, such as MD5 or SHA1.
```

Comment:
> Can we sum all these up (and other related items we're not listing here) with: "Use a vetted, tested library for password authentication" ?

Maybe; my instinct is that it's useful to know what's important to look for in a password implementation, but I may be off.

> Don't transmit passwords or session tokens over HTTP.

Think this is redundant with the HTTPS section below?